homestar-runtime/Cargo.toml

```diff
@@ -38,28 +38,29 @@ atomic_refcell = { workspace = true }
 byte-unit = { workspace = true }
 chrono = { workspace = true }
 clap = { version = "4.4", default-features = false, features = [
-  "derive",
-  "color",
-  "help",
-  "env",
-  "std",
-  "usage",
+    "derive",
+    "color",
+    "help",
+    "env",
+    "std",
+    "usage",
```

Comment:
It looks like this file may been formatted with four spaces of indentation. Is this a proposal to move to that instead of two?

Reply:
@bgins we should let the toml checker thing (taplo?) handle this. Maybe it is?

Reply:
Cargo manifest conventions suggest it should be the same indentation as Rust code: https://doc.rust-lang.org/stable/style-guide/cargo.html, so four is probably better.

Reply:
Shoot, yeah, this was my editor autoformatting using `taplo`. I can split the autoformatting out into a separate commit if you'd like.

Reply:
@QuinnWilton it's fine. As long as it as taplo for all of us!

Reply:
To @bgins below, my thought was that taplo, https://github.com/tamasfe/taplo, should handle this correctly, but I run taplo and haven't encountered these changes, so 🤷🏽.

Reply:
Ah, my mistake, I dug into this a little bit more, and it looks like what's happening is that the `taplo` LSP is honoring the `tabSize` language server setting, which is being set by my editor to 4. I'll get rid of these formatting changes.